src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -105,7 +106,22 @@ public static bool ValueDiffersFromDefault(this IParameter parameter)
          return !ValueComparer.AreValuesEqual(parameter.Value, parameter.DefaultValue.Value, CoreConstants.DOUBLE_RELATIVE_EPSILON);
       }
 
-      public static bool ParameterHasChanged(this IParameter parameter) => parameter.IsDefault == false;
+      public static bool ShouldExportToSnapshot(this IParameter parameter)
```

Comment:
I realized that this was not behaving as expected. We cannot export NaN parameters to json either